### ISSUE TITLE
JWS support is deprecated and will be removed in ItsDangerous 2.1

### ISSUE THREAD

**Issue by rsyring** [level12/keg-auth#124]
```pytb
  File "/home/rsyring/tmp/virtualenvs/kegdemo/lib/python3.9/site-packages/keg_auth/libs/authenticators.py", line 603, in on_success
    self.send_email(user)
  File "/home/rsyring/tmp/virtualenvs/kegdemo/lib/python3.9/site-packages/keg_auth/libs/authenticators.py", line 610, in send_email
    user.token_generate()
  File "/home/rsyring/tmp/virtualenvs/kegdemo/lib/python3.9/site-packages/keg_auth/model/__init__.py", line 300, in token_generate
    serializer = self.get_token_serializer(
  File "/home/rsyring/tmp/virtualenvs/kegdemo/lib/python3.9/site-packages/keg_auth/model/__init__.py", line 262, in get_token_serializer
    return itsdangerous.TimedJSONWebSignatureSerializer(
  File "/home/rsyring/tmp/virtualenvs/kegdemo/lib/python3.9/site-packages/itsdangerous/jws.py", line 201, in __init__
    super().__init__(secret_key, **kwargs)
  File "/home/rsyring/tmp/virtualenvs/kegdemo/lib/python3.9/site-packages/itsdangerous/jws.py", line 54, in __init__
    warnings.warn(
DeprecationWarning: JWS support is deprecated and will be removed in ItsDangerous 2.1. Use a dedicated JWS/JWT library such as authlib.
```

**Comment by guruofgentoo** [comment]
I started a branch on this, but this seems like it will be a little bit of a rabbit hole:
- we'll want to try to support transitional tokens. e.g. token was created for a user before the transition to the new lib, such as a verification link, and we want that to work seamlessly
- we can probably use flask-jwt-extended like we do for the JWT token auth case. The usage is a bit different, though, and I'm not sure offhand how to do the same sort of salting process that invalidates tokens when certain salt parameters change.

**Comment by guruofgentoo** [comment]
Fixed in 0.5.0
